### PR TITLE
[WIP] bpo-1294959: Better support for systems with /usr/lib64

### PR DESCRIPTION
**Pull request by stratakis**
https://bugs.python.org/issue1294959

### CODE REVIEW

**Comment by stratakis** [comment]
This PR is based on #3698 rebased on top of master and the NEWS entries tidied up a bit. I haven't tested it yet as of recently thus the WIP in the title.

File: Lib/pydoc.py [hunk 1]
```diff
@@ -390,9 +391,7 @@ def fail(self, object, name=None, *args):
 
     docmodule = docclass = docroutine = docother = docproperty = docdata = fail
 
-    def getdocloc(self, object,
-                  basedir=os.path.join(sys.base_exec_prefix, "lib",
-                                       "python%d.%d" %  sys.version_info[:2])):
+    def getdocloc(self, object, basedir=sysconfig.get_path('stdlib')):
```
Comment (merwok) [inline Lib/pydoc.py:396]: This looks like a good fix regardless of the lib/lib64/debian-triplet question!

Maybe open a distinct ticket and cherry-pick this change in a PR?

**Comment by csabella** [comment]
@stratakis, thank you for opening this to continue the work from #3698.  Please fix the merge conflicts and that will retrigger the CI builds.  Thanks!

File: Lib/pydoc.py (hunk 1 above)
Comment (vstinner) [reply Lib/pydoc.py:396]: There was already https://bugs.python.org/issue21016 issue for that and I proposed a separted PR: https://github.com/python/cpython/pull/18476

**Comment by stratakis** [comment]
Closing this in favor of #18381

**Pull request closed without merging**